kore/src/Kore/Logger/DebugProofState.hs

```diff
@@ -0,0 +1,53 @@
+{- |
+Copyright   : (c) Runtime Verification, 2019
+License     : NCSA
+-}
+
+module Kore.Logger.DebugProofState
+    ( DebugProofState (..)
+    ) where
+
+import Data.Text.Prettyprint.Doc
+    ( Pretty (..)
+    )
+import qualified Data.Text.Prettyprint.Doc as Pretty
+
+import Kore.Internal.TermLike
+    ( Variable
+    )
+import Kore.Logger
+import Kore.Step.RulePattern
+    ( ReachabilityRule (..)
+    , RewriteRule (..)
+    )
+import Kore.Strategies.ProofState
+    ( Prim (..)
+    , ProofState (..)
+    )
+
+data DebugProofState =
+    DebugProofState
+        { proofstate :: !(ProofState (ReachabilityRule Variable))
```

Comment:
```suggestion
        { proofState :: !(ProofState (ReachabilityRule Variable))
```

Reply:
It would clash with https://github.com/kframework/kore/blob/d09b5e74e0a87ba613a7ce786618f6dcfa81b801/kore/src/Kore/Strategies/ProofState.hs#L106

Reply:
Where? I.e. are you talking about a specific clash, or about the fact that the clash could occur? Usually qualified imports for record fields (as recommended by our coding standards) or importing only what's needed are enough to solve this issue, but it may not work in all cases.

Reply:
Hmm yes, you're right, sorry. I was just thinking it might be confusing to have two things named `proofState` which are both related to the same type, but after thinking about this a bit more `proofstate` might be more error-prone.